### PR TITLE
chore: Update lerna and Typescript

### PR DESCRIPTION
**Pull request by DeMoorJasper**
## What kind of change does this PR introduce?

- Update lerna
- Update TS to latest version
- Remove tsdx from csb/executors

### CODE REVIEW

File: packages/app/src/app/pages/Dashboard/Content/routes/Discover/Curate.tsx [hunk 1]
```diff
@@ -204,7 +204,10 @@ export const Collection: React.FC<CollectionTypes> = ({ album }) => {
           gap={2}
           onSubmit={event => {
             event.preventDefault();
-            updateAlbum({ id: album.id, title: event.target.title.value });
+            updateAlbum({
+              id: album.id,
+              title: (event.target as any).title.value,
```
Comment (tristandubbeld) [inline packages/app/src/app/pages/Dashboard/Content/routes/Discover/Curate.tsx:209]: We can type the submit event better later. Same goes for the submit event below.

File: packages/codesandbox-api/package.json [hunk 2]
```diff
@@ -71,7 +71,7 @@
     "@types/debug": "^0.0.30",
     "@types/glob": "^7.1.1",
     "@types/uuid": "^3.4.3",
-    "@types/node": "^10.0.0",
+    "@types/node": "^16.18.14",
```
Comment (tristandubbeld) [inline packages/codesandbox-api/package.json:74]: 🤯

File: packages/executors/package.json [hunk 3]
```diff
@@ -10,13 +11,10 @@
...
   ],
   "main": "dist/index.js",
-  "umd:main": "dist/codesandboxexecutors.umd.production.js",
-  "module": "dist/codesandboxexecutors.es.production.js",
   "typings": "dist/index.d.ts",
   "scripts": {
-    "build": "tsdx build",
-    "build:dev": "yarn build",
-    "start": "tsdx watch"
+    "build": "tsc --outDir dist",
```
Comment (tristandubbeld) [inline packages/executors/package.json:16]: Awesome!

**Review by tristandubbeld — COMMENTED** [review]
Awesome job! Made one comment for myself to address something later (not in this PR) but other than that everything looks good. Can we try deploying this to stream and test it there?

**Comment by DeMoorJasper** [comment]
@tristandubbeld just deployed it to staging, everything still appears to be working :D

**Pull request merged**